earth_enterprise/rpms/build.gradle

```diff
@@ -1,17 +1,26 @@
-/*
-    See documentation on the osPackage plugin here: https://github.com/nebula-plugins/gradle-ospackage-plugin
+// Command-line parameters:
+//
+//     ./gradlew -PbuildOpenGee:
+// Compile and install Open GEE in a temporary directory before packaging.
+// By default no compilation, or installation are run.  In that case, you must
```

Comment:
This line is confusing to me.  The previous line says that this command compiles and installs Open GEE, and this line says that it doesn't.  I think maybe most of this paragraph is meant to apply to the case when you don't specify `-PbuildOpenGEE`, but that wasn't really clear to me.